cypress/integration/TextInputLabs.spec.ts

```diff
@@ -0,0 +1,79 @@
+import * as h from '../helpers';
```

Comment:
These are copied from other Text Input, as a follow up we should identify additional tests, e.g. that the model applies ids, for, and aria-* correctly.